extra/redisotel/config.go

```diff
@@ -63,17 +62,11 @@ func newConfig(opts ...baseOption) *config {
 		opt.apply(conf)
 	}
 
-	conf.attrs = append(conf.attrs, semconv.DBSystemKey.String(conf.dbSystem))
+	conf.attrs = append(conf.attrs, semconv.DBSystemRedis)
 
 	return conf
 }
 
-func WithDBSystem(dbSystem string) Option {
-	return option(func(conf *config) {
-		conf.dbSystem = dbSystem
-	})
-}
```

Comment:
@esara could you please restore this? It is a public API and it is still useful for those who use this client with other systems that provide Redis compatible API.

Reply:
of course, sorry for not realizing, restored